qt/applications/workbench/workbench/plotting/figureinteraction.py

```diff
@@ -136,6 +136,15 @@ def on_scroll(self, event):
             and len(event.inaxes.lines) == 0
         ):
             return
+
+        """
+        Disable the legend's draggability when the scroll event is
+        enabled to prevent the legend from unintentionally moving,
+        which is the default behavior in Matplotlib.
+        """
```

Comment:
These kinds of triple quotes comments should be used only at the beginning of functions, and they're called docstrings:  https://peps.python.org/pep-0257/. 
Instead you can make a standard comment:
```suggestion
# Correcting for picking legend with scroll event, which is default in Matplotlib
```